### ISSUE TITLE
GraphQL returns Unregistered Type error when running query. v2.0 Beta

### ISSUE THREAD

**Issue by simeon-smith** [sebastian-lenz/craft-linkfield#81]
GraphQL is returning an Unregistered Type error when running a query in Craft 3.3.13.

Query:
```
    fragment SeomaticData on SeomaticInterface {
    metaTitleContainer
    metaTagContainer
    metaLinkContainer
    metaScriptContainer
    metaJsonLdContainer
  }
  
  query EDMOGlobals {
    globalSets {
      ...on edmoBusinessInfo_GlobalSet {
        edmoAddress
        googleMapsiFrame
        googleMapsLink
        # { This doesn't currently work. Craft 3.3.13 - Typed link field 1.0.19
          #   url
          #   text
          #   target
          # }
        edmoPhone
        edmoFax
        gsaContractInfo {
          columnHeading
          columnInfo
        }
      }
      ...on edmoMission_GlobalSet {
        edmoMissionTaglineHeading
        edmoMissionTaglineSubheading
        edmoMissionBackground {
          id
          url
          title
          height
          width
        }
        edmoMissionLetters {
          ...on edmoMissionLetters_BlockType {
            id
          heading
          copy
          }
          
        }
      }
      ...on termsConditions_GlobalSet {

        termsConditions
      }
      
    }
    seomatic(asArray: true) {
      ...SeomaticData
    }
    about: entries(section: "about") {
      id
      slug
      title
    }
    customerService: entries(section: "customerService") {
      id
      slug
      title
    }
    services: entries(section: "services") {
      id
      slug
      title
    }
  }
```

**Comment by magnus-bergman** [comment]
Looks like this might have been fixed in https://github.com/sebastian-lenz/craft-linkfield/commit/2a7318c986366059026b28a077fb353053a1abff?

**Comment by magnus-bergman** [comment]
Can confirm that the fix in the commit referenced in the previous comment works. When is the next beta release planned @sebastian-lenz ?

**Comment by sebastian-lenz** [comment]
Hi and thanks for reporting this. Unfortunately the v1 version of the plugin has no native GraphQL support, however, it comes with support for CraftQL. We have a v2 branch which comes with native GraphQL support (but no CraftQL support), however, we don't have yet solved the update path completely (see #77). There is a patch for the upgrade issues (see https://github.com/sebastian-lenz/craft-linkfield/commit/8f935fb208d7185ffbfa4236436254a8a9b09559) but as testing it is quite time consuming and I wasn't able to squeeze it into my time schedule it's currently unreleased.

**Comment by magnus-bergman** [comment]
Totally understand! It's a major version afterall, so breaking changes are to be expected I guess?

In anycase, I'm setting up a new project which is kind of experimental and I removed all link fields created with 1.0 in order to get the upgrade to beta to work. Would it be possible to release a 2.0.0-beta.7 with the fix so that the queries work? Right now the beta doesn't work at all.

**Comment by Jones-S** [comment]
This is resolved in `2.0.x-dev` as it seems. I just used this version in my `composer.json` and I got my graphql to work without throwing `Tried to load an unregistered type „linkField_Link”.`

**Comment by sebastian-lenz** [comment]
Okay, I'll close this due to inactivity. Please feel free to open a new issue should you run into any problems.
